### PR TITLE
Multi tables

### PR DESCRIPTION
**Pull request by yosefHesham**

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -11,3 +11,47 @@ INSERT INTO animals (name, date_of_birth, escape_attemps, neutrued, weight_kg) V
...
+JOIN species ON animals.species_id = species.id 
+GROUP BY species.id;
+
+Select * from animals join owners on owners.full_name = 'Dean Winchester' where escape_attemps = 0;
+
+SELECT COUNT(*), owners.full_name AS owner FROM animals 
+JOIN owners ON animals.owner_id = owners.id 
+GROUP BY owners.full_name 
+ORDER BY COUNT(*) DESC 
+LIMIT 1;
```
Comment (Lordkaito) [inline data.sql:57]: - [ ] Please make sure to include this code into your `queries.sql` file, as this is not part of the data.

File: data.sql [hunk 2]
```diff
@@ -11,3 +11,47 @@ INSERT INTO animals (name, date_of_birth, escape_attemps, neutrued, weight_kg) V
...
+
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith') WHERE name = 'Agumon';
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell') WHERE name = 'Gabumon' OR name = 'Pikachu';
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') WHERE name = 'Devimon' OR name = 'Plantmon';
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') WHERE name = 'Angemon' OR name = 'Boarmon';
+
+
+select * from animals join owners on owners.id = owner_id;
```
Comment (Lordkaito) [inline data.sql:40]: - [ ] I believe this query is supposed to select all animals that belong to Melody Pond, so I think you are missing a `WHERE` clause, so please make sure to add it.

File: data.sql [hunk 3]
```diff
@@ -11,3 +11,47 @@ INSERT INTO animals (name, date_of_birth, escape_attemps, neutrued, weight_kg) V
...
+
+Select * from animals join species on species.id = species_id and species.name = 'Pokemon';
+Select * from animals right outer join owners on owners.id = owner_id;
+
+
+SELECT COUNT(*), species.name 
+FROM animals 
+JOIN species ON animals.species_id = species.id 
+GROUP BY species.id;
+
```
Comment (Lordkaito) [inline data.sql:50]: - [ ] I think you are missing a query here, which is supposed to list all Digimon owned by Jennifer Orwell, so please make sure to implement it.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @yosefHesham, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @yosefHesham    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**